lib/reek/cli/options.rb

```diff
@@ -9,9 +9,14 @@ module CLI
     #
     # See {file:docs/Command-Line-Options.md} for details.
     #
-    # :reek:TooManyInstanceVariables: { max_instance_variables: 7 }
+    # :reek:TooManyInstanceVariables: { max_instance_variables: 9 }
     # :reek:Attribute: { enabled: false }
+    #
     class Options
+      DEFAULT_SUCCESS_EXIT_CODE  = 0
+      DEFAULT_ERROR_EXIT_CODE    = 1
```

Comment:
DEFAULT_ERROR_EXIT_CODE is **not** configurable at the moment. I just put it here for consistency reasons. 
We exit the hard way using this code only at 2 places:
1.) https://github.com/troessner/reek/pull/808/files#diff-8cb50151ad3a458ecb02fe222bb330eaR52
2.) https://github.com/troessner/reek/pull/808/files#diff-8cb50151ad3a458ecb02fe222bb330eaR45

and that's where we're actually parsing options. So making them configurable right now would be completely pointless.
If we start to introduce this error code across then we should make it configurable as well.
And we should probably. Thinking about #809